requirements.txt

```diff
@@ -1,4 +1,4 @@
-ase>=3.16
+git+https://gitlab.com/ase/ase.git@database_writemany#egg=ase-3.16.3b1
```

Comment:
I am not keen on this. Are we sure every aspect of these changes are stable? This also separates CatKit from the standard package and may introduce unknown pip installation complexity?

Reply:
Jens J is going to merge the changes into master soon (I hope!). So this is just a temporary fix. I know it's not optimal...

Reply:
I also hope that this is quite temporary. I do not want to lose compatibility with ASE upstream. But in the meantime, it would behoof us if we could move forward with developing upload capability. So I would take @kirstenwinther  branch but keep a close eye on ASE master and move back as soon as possible.